src/DynamoCoreWpf/Controls/HeaderStripItem.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Dynamo.UI.Controls
```

Comment:
This file can be deleted and move the class alongsdie ```HeaderStrip``` control. It's just too small.